tests/Doctrine/Tests/DbalTypes/Issue8565EmployeePayloadType.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DbalTypes;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\JsonType;
+use Exception;
+
+class Issue8565EmployeePayloadType extends JsonType
```

Comment:
We usually use `GH` instead of `Issue` here. There is no guarantee that we aren't migrate to something else than Github in the future and that's a risk of collision.

Reply:
Fixed